### ISSUE TITLE
Don't inherit nomination time from delay-rejected versions

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#19439]
### Describe the problem and steps to reproduce it:

In the reviewer tools, the nomination time is inherited from previous versions if they have not been reviewed. It seems we never updated the logic when we introduced delayed rejections?

### What happened?
When a version is delay-rejected, the nomination time, should be reset.

### What did you expect to happen?
As far as I can see, when a version is delay-rejected, the nomination time, subsequent submissions inherit the nomination time from the delay-reject version.

### Anything else we should know?
(Please include a link to the page, screenshots and any relevant files.)

**Comment by wagnerand** [comment]
If updating the inheriting turns out to be too complex, we can also consider removing inheritance altogether. Let's chat if you believe we should do that.

**Comment by ioanarusiczki** [comment]
@wagnerand 

I submitted some new versions and rejected them with a delay -> each has it's individual rejection period set and it seems to be exactly what I've chosen at rejection.  [Example](https://reviewers.addons-dev.allizom.org/en-US/reviewers/review-unlisted/621769#review-actions) 

I'm not sure what "nomination time" is so could you add for qa some str?

**Comment by diox** [comment]
Nomination time is the date & time that the version started being reviewable - essentially the submission time. However, if you already have a version waiting for review and you submit another, the new version nomination time is inherited from the older one. It is then used in reviewer tools queues that display the "Waiting time": it's the time the developer had to wait for a review.

Previously, in that scenario, if the first version was delay-rejected, the new version would still inherit from that first nomination time. Now, delay-rejected versions will be ignored for the purpose of inheriting the nomination time.

**Comment by ioanarusiczki** [comment]
@diox Ah, thanks! I went in a wrong direction - looking for clues on the reviewer pages, but the waiting time si displayed into the queues. 

One of them is the recommended queue. I submitted a version and rejected it with delay then submitted a new version and went back to check the waiting time -> on dev the waiting time is no longer inherited from the previous version (because I see seconds) . On stage the time is clearly longer - some minutes ago when I submitted the delayed version. 

I'm going to do some more testing around it tomorrow morning.

**Comment by ioanarusiczki** [comment]
@diox @wagnerand 

I applied the scenario described above and also checked the other queues with "Waiting Time":

[Other Pending Review](https://reviewers.addons-dev.allizom.org/en-US/reviewers/queue/extension) : there's no time displayed for the version submitted after the one pending 

![waiting time](https://user-images.githubusercontent.com/33448286/179953508-9e7a195c-d644-4501-a35d-1ee3216195aa.png)

same for the[ New Themes ](https://reviewers.addons-dev.allizom.org/en-US/reviewers/queue/theme_new )queue (right now the first result)

The theme[ Updates](https://reviewers.addons-dev.allizom.org/en-US/reviewers/queue/theme_updates) seemed to display the waiting time correctly after I submitted [version 3](https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/621271) which is the first version after the delayed one. Next versions 4 and 5 seem to inherit the nomination time from v3 if this one has not been rejected or delayed rejected. I see the same behavior for the recommended queue. - could be the expected

**Comment by diox** [comment]
Yes, we continue to inherit nomination from versions not yet reviewed & not under a delayed rejection.

**Comment by ioanarusiczki** [comment]
@diox
 I also tested for the recommended and theme updates queue with rejected immediately -> the next version submitted is not inheriting the nomination time. 

What I'm not sure of is the pending review and new themes queue - not showing time at all. Is that ok?

**Comment by wagnerand** [comment]
Looking at the 

> @diox @wagnerand
> 
> I applied the scenario described above and also checked the other queues with "Waiting Time":
> 
> [Other Pending Review](https://reviewers.addons-dev.allizom.org/en-US/reviewers/queue/extension) : there's no time displayed for the version submitted after the one pending
> 
> ![waiting time](https://user-images.githubusercontent.com/33448286/179953508-9e7a195c-d644-4501-a35d-1ee3216195aa.png)

I looked at the extensions in that queue on dev and it looks like a bug to me. The nomination time should be the one of the latest (currently pending) version, instead of `None`.

**Comment by ioanarusiczki** [comment]
@wagnerand Ok, thanks for confirming it, I'll open a followup.
